### ISSUE TITLE
Error when setting color-scheme

### ISSUE THREAD

**Issue by jgollenz** [ellisonleao/gruvbox.nvim#100]
I get this after installing with packer (`use {'ellisonleao/gruvbox.nvim'}`) and setting it with `:color gruvbox` :

```
Error detected while processing /home/nope/.local/share/nvim/site/pack/packer/start/gruvbox.nvim/colors/gruvbox.lua:
E5113: Error while calling lua chunk: ...m/site/pack/packer/start/gruvbox.nvim/colors/gruvbox.lua:5: attempt to call field 'load' (a nil value)
stack traceback:
        ...m/site/pack/packer/start/gruvbox.nvim/colors/gruvbox.lua:5: in main chunk
```
Uncommenting that line results in no error but also does not set the color-scheme. I had this issue before the switch from Lush, so I used [this](https://github.com/eddyekofo94/gruvbox-flat.nvim) as a replacement for a while. Hoped it was a Lush issue, apparently not.

Neovim Version: 0.6.0

**Comment by ellisonleao** [comment]
hey @jgollenz do you have some configs we can check?  it's very strange because i am using the same version

**Comment by jgollenz** [comment]
What configs do you mean specifically? I have other plugins enabled and I could try if turning them all off changes the behavior

**Comment by jgollenz** [comment]
Ok, the error goes away when I disable  eddyekofo94/gruvbox-flat.nvim, thats good. But I still don't get the expected behavior from gruvbox.nvim. It works for my Lualine (which is disabled in the gif), but I assume this is hardcoded into the Lualine plugin itself

![gruvbox](https://user-images.githubusercontent.com/12050098/150938111-2695bce2-73cf-42ba-809d-d89e201703c2.gif)

**Comment by ellisonleao** [comment]
@jgollenz i am asking you lua configs because it seems you have 2 gruvbox plugins installed AFAIK right? Because if you do, there will be issues on that function because both namespaces are `gruvbox`

**Comment by jgollenz** [comment]
Indeed, although one is called `gruvbox-flat` and should not collide in my opinion. Anyway, your are right, they do. But removing `gruvbox-flat` still does not give me a working `gruvbox` plugin. Where would I find the lua configs you mean?

**Comment by ellisonleao** [comment]
Can you share your editor configs? Are you defining `termguicolors` setting?

**Comment by jgollenz** [comment]
Thank you! I did not enable `termguicolors`, that seems to have been the issue. But I did not know that this was necessary, maybe it should be added to prerequisites?

**Comment by ellisonleao** [comment]
yeah, I will add it to the README. Closing this one

**Comment by twhiting** [comment]
I also ran into this issue where `gruvbox-flat` conflicted with this theme. Perhaps it is worth documenting in the readme if it makes sense? It is likely pretty rare though so nbd.
